app/views/docs/upgrade.phtml

```diff
@@ -1,19 +1,40 @@
-<p>To upgrade your Appwrite server from an older version, you should use the Appwrite migration tool <b>after you have installed the new version</b>. The migration tool will adjust your Appwrite data to the new version structure to make sure your Appwirte data is compatible with any internal changes.</p>
-
-<p>Make sure to run the migration tool <b>after</b> you already finished <a href="/docs/installation">installing</a> the new version. When you install the new version, you need to run the installation command from the parent directory of your `appwrite` directory where your docker-compose.yml is located or to manually update your docker-compose.yml file. Your `appwrite` directory will be located at the location of the working dir you used while setting Appwrite in the first place.</p>
+<p>To upgrade your Appwrite server from an older version, you should use the Appwrite migration tool <b>after you have installed the new version</b>. The migration tool will adjust your Appwrite data to the new version structure to make sure your Appwrite data is compatible with any internal changes.</p>
 
 <div class="notice margin-bottom">
     <h3>A Note About Migration</h3>
-    
+
     <p>It is highly recommended to <a href="https://medium.com/sunkay-imho/docker-data-volumes-and-data-volume-containers-use-cases-abde84a10317" target="_blank" rel="noopener">backup your server</a> data and your <b>docker-compose.yml</b> file settings before running the migration tool. It is recommended to run the migration process on a dev instance and make sure your application is working well and that you have checked for any breaking changes in the new version <a href="https://github.com/appwrite/appwrite/blob/master/CHANGES.md" target="blank" rel="noopener">changelog.</a></p>
 </div>
 
 <!-- <p>When updating to a new Appwrite version, make sure to run the <a href="/docs/installation">installation script for the new version</a> from the same location, you executed your original setup. This will allow the installation script to update your existing docker-compose.yml file. The correct location for execution is the parent directory of your existing appwrite directory.</p> -->
 
 <!-- <p>The data migration tool will allow you to easily migrate your current Appwrite data to work with the new version. </p> -->
 
-<p>To start the migration, run the following command from your terminal once your backup is ready and that your new Appwrite setup has been completed successfully. Replace "[APPWRITE_CONTAINER_ID]" with your container ID. You can find your container ID by running `docker ps` command.</p>
+<p> Head over to the directory containing your <b>docker-compose.yml</b> . If Appwrite is currently running, your first step would be to shut it down using: </p>
+<div class="ide margin-bottom" data-lang="bash" data-lang-label="Bash">
```

Comment:
Why do we need to shut it down before setup? The install script already takes care of that

Reply:
Okay I'll remove this